crates/cxx-qt-gen/src/parser/property.rs

```diff
@@ -3,14 +3,86 @@
 //
 // SPDX-License-Identifier: MIT OR Apache-2.0
 
-use syn::{Ident, Type, Visibility};
+use syn::{parse::ParseStream, Attribute, Ident, Result, Token, Type};
 
 /// Describes a single Q_PROPERTY for a struct
 pub struct ParsedQProperty {
     /// The [syn::Ident] of the property
     pub ident: Ident,
     /// The [syn::Type] of the property
     pub ty: Type,
-    /// The [syn::Visibility] of the property
-    pub vis: Visibility,
```

Comment:
As `#[qproperty(T, NAME)]` is supposed to be a shorthand for `#[qproperty(T, NAME, read = Self::my_getter, write = Self::my_setter)]`, to me it seems that the shorthand could just always generate as `pub` visibility then custom stuff can do anything else.

Otherwise we'd need the API to be `#[qproperty(pub, T, NAME)]` or `#[qproperty(pub T, NAME)]` `#[qproperty(T, pub NAME)]` or something :thinking:

Reply:
Moved back to question on #555 as keeping autogen pub makes sense i think.